specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/ManagedEnvironments.json

```diff
@@ -889,6 +889,297 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "managedHttpRouteEnvelope",
+            "in": "body",
+            "description": "Http Route config to be created or updated",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/HttpRouteConfig"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/HttpRouteConfig"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/HttpRouteConfig"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update Http Route": {
+            "$ref": "./examples/ManagedHttpRoutes_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ManagedEnvironments",
+          "ManagedHttpRoutes"
+        ],
+        "summary": "Update tags of a manged http route object",
+        "description": "Patches an http route config resource. Only patching of tags is supported",
+        "operationId": "ManagedHttpRoutes_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "httpRouteName",
+            "in": "path",
+            "description": "Name of the Http Route Config Resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "managedHttpRouteEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Properties of http route config that need to be updated",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/ManagedHttpRoutePatch"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
No since there is no compute/ background work required besides persisting CRD in cluster and envoy reconciling (which is very quick).
GET,PUT,PATCH,DELETE should all be near real time operation.